WeatherApp/Models/DailyWeather/ApiModels/WeeklyWeatherResponse.swift

```diff
@@ -10,16 +10,16 @@ import Foundation
 
 struct WeeklyWeatherResponse: Decodable {
     
-    let dailyForecast: [DailyForecast]
-    let hourlyForecast: [HourlyForecast]
     let latitude: Double
     let longitude: Double
+    let dailyForecast: [DailyForecast]
+    let hourlyForecast: [HourlyForecast]
```

Comment:
I mentioned before that you should rename either the DailyForecast class to something like DailyForecastItem or rename the attributes in a way that reflects they are an array. 
You have an attribute _dailyForecast_ that is an array `[DailyForecast]` and then in other classes you also have _dailyForecast_ variable that is a single instance of `DailyForecast`.